src/wordCount/Main.java

```diff
@@ -0,0 +1,63 @@
+
+        // System.out.println(text);
+
+        String text = text2.toLowerCase();
+
+        text.replaceAll("[[\\.\\?\\!\\,\\;\\:\\{\\}\\(\\)\\']]", "");
+        String[] words = text.split(" +");
+
```

Comment:
Line 14 - Make sure to assign the result of the `.replaceAll` method to the `text` variable - it returns a String value but does not mutate the original String, which is why you'll still have to assign it like so:
```
text = text.replaceAll("[[\\.\\?\\!\\,\\;\\:\\{\\}\\(\\)\\']]", "");
```

Take note that **not assigning the return value will cause the punctuation marks to be retained, potentially altering the results given when running your code**.

You can also chain it to your use of the `.toLowerCase` method, like so:
```
text= text.toLowerCase().replaceAll("[[\\.\\?\\!\\,\\;\\:\\{\\}\\(\\)\\']]", "");
```